### PR TITLE
Allow user to withdraw in either native or pseudo native asset address.

### PR DESCRIPTION
**Pull request by pRoy24**
Resolves
https://github.com/protofire/omen-exchange/issues/1538
https://github.com/protofire/omen-exchange/issues/1585

When interacting with a market in ETH, SPOA or xDai you should be able to withdraw you pool/outcome tokens in either native asset or wrapped asset.

Tested on ETH and Sokol markets.

### CODE REVIEW

**Review by hexyls — APPROVED** [review]
awesome! 🎉

**Review by pimato — CHANGES_REQUESTED** [review]
Hey @pRoy24 great work! Can we keep the pool statistic always in base token?
Currently, if I switch to a wrapped token the pool statistic is getting transormed too:
Bildschirmfoto 2021-02-01 um 12 33 22

**Comment by pRoy24** [comment]
> Hey @pRoy24 great work! Can we keep the pool statistic always in base token?
> Currently, if I switch to a wrapped token the pool statistic is getting transormed too:
> Bildschirmfoto 2021-02-01 um 12 33 22

Resolved.

**Review by Mi-Lan — CHANGES_REQUESTED** [review]
- Modal that pops up needs to be updated it shows withdrawing eth when it should be weth 
Screenshot 2021-02-02 at 17 16 30

**Review by Mi-Lan — CHANGES_REQUESTED** [review]
- When on ETH market and if i deposit in WETH i don't see option to choose in which to withdraw...
Screenshot 2021-02-02 at 17 41 04
- Here is market i tested it on http://localhost:3000/#/0xb4277c368b817db02bab8b684c9bfe10200461f7

**Review by pimato — CHANGES_REQUESTED** [review]
Getting

```
./src/components/market/sections/market_pooling/market_pool_liquidity.tsx
  Line 598:16:  Parsing error: '}' expected```

**Comment by pimato** [comment]
Hover styling should be for the text and icon, currently only text has hover styling:
Bildschirmfoto 2021-02-03 um 09 18 16

**Comment by pimato** [comment]
It looks like the switch does not work sometimes:

1. Deposit wrapped xdai here: http://localhost:3000/#/0x8766bb7aa3e657b284480e2679fa1f479744f9b7
2. Try to withdraw and switch to wrapped xdai --> BUG: Nothing happens:
Bildschirmfoto 2021-02-03 um 09 19 47

**Review by pimato — CHANGES_REQUESTED** [review]
see comments

**Comment by pRoy24** [comment]
> It looks like the switch does not work sometimes:
> 
> 1. Deposit wrapped xdai here: http://localhost:3000/#/0x8766bb7aa3e657b284480e2679fa1f479744f9b7
> 2. Try to withdraw and switch to wrapped xdai --> BUG: Nothing happens:
> 
> Bildschirmfoto 2021-02-03 um 09 19 47

Yes good catch, the collateral is shared between deposit and withdraw tabs and if you change to wrapped asset in deposit the check fails in withdraw. Fixed

**Comment by pimato** [comment]
Hey @pRoy24 could you use this svg:
[swap-asset.svg.zip](https://github.com/protofire/omen-exchange/files/5917754/swap-asset.svg.zip)

**Review by pimato — CHANGES_REQUESTED** [review]
replace svg

**Comment by pimato** [comment]
[](https://gifyu.com/image/UDxS)

File: app/src/components/market/common/switch_transaction_token/index.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      path {
+        fill: ${props => props.theme.colors.primaryLight};
+      }
+    }
+  }
+`
+
+const SwitchComponentText = styled.div`
+  display: inline-block;
+  margin-right: 10px;
```
Comment (pimato) [inline app/src/components/market/common/switch_transaction_token/index.tsx:29]: lets make this margin-right: 8px;

**Review by pimato — CHANGES_REQUESTED** [review]
can we get the icon vertical centered?
Bildschirmfoto 2021-02-03 um 14 55 12

**Review by pimato — APPROVED** [review]

File: app/src/util/tools.ts [hunk 2]
```diff
@@ -472,3 +472,16 @@ export const onChangeMarketCurrency = (
...
+ * If collateral is cToken type then display is the base collateral
+ * Else display is the collateral
+ */
+export const getInitialCollateral = (networkId: number, collateral: Token): Token => {
+  if (collateral.address === getWrapToken(networkId).address) {
+    return getNativeAsset(networkId)
+  } else {
+    return collateral
+  }
+}
```
Comment (KadenZipfel) [inline app/src/util/tools.ts:487]: We should have a test for this in `tools.spec.ts`, same for all other tools functions

**Review by KadenZipfel — CHANGES_REQUESTED** [review]
Overall great work!

**Review by Mi-Lan — APPROVED** [review]

**Review by KadenZipfel — APPROVED** [review]

**Pull request merged**